codec-http/src/main/java/io/netty/handler/codec/http/HttpHeadersFactory.java

```diff
@@ -18,12 +18,12 @@
 /**
  * An interface for creating {@link HttpHeaders} instances.
  * <p>
- * The default implementation is {@link HttpHeadersBuilder},
- * and the default instance is {@link HttpHeaders#DEFAULT_HEADER_FACTORY}.
+ * The default implementation is {@link DefaultHttpHeadersFactory},
+ * and the default instance is {@link DefaultHttpHeadersFactory#headersFactory()}.
  */
 public interface HttpHeadersFactory {
     /**
      * Create a new {@link HttpHeaders} instance.
      */
-    HttpHeaders createHeaders();
+    HttpHeaders newHeaders();
```

Comment:
Follow-up for the previous thread: https://github.com/netty/netty/pull/13609#discussion_r1390310472
Do you plan to add a 2nd method on this interface to avoid "functional"?

Reply:
I was still debating with myself about that, but I guess yes.